x-pack/plugins/maps/public/connected_components/toolbar_overlay/toolbar_overlay.tsx

```diff
@@ -56,6 +57,10 @@ export function ToolbarOverlay(props: Props) {
       </EuiFlexItem>
 
       {renderToolsControl()}
+
+      <EuiFlexItem>
```

Comment:
I would hide this toggle if there are no time-aware layers on the map. similar to how we hide filter-bar, or draw tools when layers are not drawable/filterable